src/components/RedLiner/RedLiner.mdx

```diff
@@ -65,4 +65,34 @@ Use the prop `showOnHover` to hide RedLiner until the child component is hovered
   <RedLiner config={{ color: 'purple', displayOpts: ['height', 'info'], infoOpts: ['borderRadius', 'color'] }} showOnHover>
     <button style={buttonStyle}>Measure Me</button>
   </RedLiner>
-</Playground>
\ No newline at end of file
+</Playground>
+
+### Components Prop
+
+It's possible to replace and modify certain components in RedLiner. In this example, we replace the default InfoBox component with a custom version.
+
+<Playground>
+  <RedLiner components={{ InfoBox: CustomInfoBox }}>
```

Comment:
Custom component passed in 

Preview: 
https://deploy-preview-19--redliner.netlify.com/redliner